src/pages/CookieClicker.vue

```diff
@@ -70,63 +70,65 @@ const cps = computed(() =>
 );
 
 setInterval(() => {
-  cookies.value += cps.value;
+  cookies.value += cps.value / 10;
```

Comment:
okay we do this 10 times more so give 10 times less cookies but why? we could also do this every 1 ms and give 1000 times less cookies? Can you come up with idea why it would be bad idea?